proposals/lip-0037.md

```diff
@@ -33,23 +33,25 @@ Replay attacks for transactions and blocks, i.e. replaying transactions or block
 
 Note that any updates to the serialization specification of a message type must be accompanied by introducing a new tag. For example, when the [transaction schema](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0028.md#transactionschema-schema) is updated, a new tag like `"LSK_TX:V2_"` must be defined and used.
 
-### Network Identifiers
+### Chain Identifiers
 
-Network identifiers for transaction signatures and block signatures were already introduced in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) and [LIP 0024](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0024.md#update-to-the-block-header-signing-procedure) respectively to prevent replay attacks on other chains. Here, we specify how to use a network identifier for any kind of message to prevent replay attacks on other chains in general.
+Chain identifiers for transaction signatures and block signatures were already introduced in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) and [LIP 0024](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0024.md#update-to-the-block-header-signing-procedure) (as network identifiers) to prevent replay attacks on other chains. Here, we specify how to use a chain identifier for any kind of message to prevent replay attacks on other chains in general.
+
+Chain identifiers are 4-byte values that follow a specific format: the first byte is used to identify the network in which the chain is running (either the mainnet, official testnet, or any other test network); the other 3 bytes identify the chain within the network. We include the network-specific prefix explicitly to ensure that a chain does not use the same chain identifier in the test network as in the mainnet.
 
 ## Specification
 
 ### Message Tags
 
-Every binary message must be tagged before being signed. This is done as specified by the function `tagMessage` in the pseudo code below: A specific ASCII-encoded tag and a network identifier as specified in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) are prepended to the message.
+Every binary message must be tagged before being signed. This is done as specified by the function `tagMessage` in the pseudo code below: A specific ASCII-encoded tag and a chain identifier as specified in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) are prepended to the message.
```

Comment:
The chain identifier should be specified in this LIP and is different compared to the network identifier in LIP 9 which is defined as `network identifier = H(nethash + community identifier)`.

Reply:
Do you think I have to write this explicitly? It is somehow obvious that here we are given the new specs

Reply:
```suggestion
Every binary message must be tagged before being signed. This is done as specified by the function `tagMessage` in the pseudo code below: A specific ASCII-encoded tag and a chain identifier are prepended to the message.
```

Reply:
I just wanted to say the reference to LIP 9 here is rather confusing then helpful so I would just remove it.